webpack/assets/javascripts/react_app/components/HostDetails/ExperimentalAlert.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import {
+  Alert,
+  Bullseye,
+  AlertActionCloseButton,
+  AlertActionLink,
+} from '@patternfly/react-core';
+import { translate as __ } from '../../common/I18n';
+import { visit } from '../../../foreman_navigation';
+
+const ExperimentalAlert = () => {
+  const [alertVisibility, setAlertVisibility] = useState(true);
+  if (!alertVisibility) return null;
+  return (
+    <Bullseye>
+      <Alert
+        variant="info"
+        title={__('This page is an experimental feature')}
+        actionClose={
+          <AlertActionCloseButton onClose={() => setAlertVisibility(false)} />
+        }
+        actionLinks={
+          <>
+            <AlertActionLink
+              onClick={() =>
+                visit(
+                  'https://community.theforeman.org/t/foreman-3-0-new-host-detail-page-feedback/25281'
+                )
+              }
+            >
+              {__('Share feedback')}
+            </AlertActionLink>
+          </>
+        }
+      >
+        <p>
+          {__(
+            'What do you think of new design? Send us your thoughts so we can continue to improve this page.'
```

Comment:
```suggestion
            'What do you think of the new design? Send us your thoughts so we can continue to improve this page.'
```
I'm an idiot. Sorry @amirfefer - I caught Czechlish.

Reply:
fixed :)

Reply:
Thank you  !